src/OpenRiaServices.DomainServices.Tools/Test/CreateRiaClientFilesTaskTests.cs

```diff
@@ -361,8 +396,11 @@ public void CreateRiaClientFiles_Validate_Generated_Files()
                 {
                     fileListContents = t1.ReadToEnd();
                 }
-                Assert.IsTrue(fileListContents.Contains("ServerClassLib.g.cs"), "Expected file list to have ServerClassLib.g.cs but instead had " + fileListContents);
-                Assert.IsTrue(fileListContents.Contains("TestEntity.shared.cs"), "Expected file list to have TestEntity.shared.cs but instead had " + fileListContents);
+
+                // Files list 
+                bool shouldCopy = (sharedFilesMode == OpenRiaSharedFilesMode.Copy);
+                foreach (var sharedFile in expectedSharedFiles)
+                    Assert.AreEqual(shouldCopy, fileListContents.Contains(sharedFile), "Checking if OpenRiaFiles.txt contains '{0}' with mode '{1}' actual content is '{2}'", sharedFile, sharedFilesMode, fileListContents);
```

Comment:
Use  `CollectionAssert.IsSubsetOf`   , `IsNotSubsetOf`  might yield more readable code?

Reply:
`CollectionAssert.Contains` probably been the best alternative, but I wanted to change as little as possible in the unit tests so I kept the existing logic as much as possible